### PR TITLE
Reword the different ways to install reproc

### PR DESCRIPTION
**Pull request by jetm**
Add the instruction to install from Arch Linux and adjust items in the TOC.

#3

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -59,11 +62,22 @@ either make an issue or ask questions directly in the reproc
...
+Below it's explained all possible options.
 
-### FetchContent
+### Install from a Linux binary package
+
+#### Arch Linux
+If you're using Arch Linux or any distro derived from it, you can install
+reproc with:
+```sh
+yay -S reproc
```
[the remark this answers is not in the document]
Comment (jetm) [reply README.md]: Done

File: README.md [hunk 2]
```diff
@@ -59,11 +62,22 @@ either make an issue or ask questions directly in the reproc
 
 ## Getting started
 
-To use reproc you'll have to compile it first. reproc can either be installed or
-built as part of your project when using CMake. We explain all possible options
-below.
+reproc can either be installed from a binary package or built as part of your
+project when using CMake. Latter option will always have to compile it first.
+Below it's explained all possible options.
```
[the remark this answers is not in the document]
Comment (jetm) [reply README.md]: Done

File: README.md [hunk 3]
```diff
@@ -59,11 +62,22 @@ either make an issue or ask questions directly in the reproc
 
 ## Getting started
 
-To use reproc you'll have to compile it first. reproc can either be installed or
-built as part of your project when using CMake. We explain all possible options
-below.
+reproc can either be installed from a binary package or built as part of your
+project when using CMake. Latter option will always have to compile it first.
```
[the remark this answers is not in the document]
Comment (jetm) [reply README.md]: Done

File: README.md [hunk 4]
```diff
@@ -8,10 +8,13 @@
 - [Features](#features)
 - [Questions](#questions)
 - [Getting started](#getting-started)
-  - [FetchContent](#fetchcontent)
-  - [Git submodule/vendor](#git-submodulevendor)
-  - [Install](#install)
-  - [CMake options](#cmake-options)
+  - [Install from a Linux binary package](#install-from-a-linux-binary-package)
+    - [Arch Linux](#arch-linux)
+  - [Build as part of your project when using CMake](#build-as-part-of-your-project-when-using-cmake)
```
[the remark this answers is not in the document]
Comment (jetm) [reply README.md]: Done

File: README.md [hunk 5]
```diff
@@ -8,10 +8,13 @@
 - [Features](#features)
 - [Questions](#questions)
 - [Getting started](#getting-started)
-  - [FetchContent](#fetchcontent)
-  - [Git submodule/vendor](#git-submodulevendor)
-  - [Install](#install)
-  - [CMake options](#cmake-options)
+  - [Install from a Linux binary package](#install-from-a-linux-binary-package)
```
[the remark this answers is not in the document]
Comment (jetm) [reply README.md]: Done

**Comment by DaanDeMeyer** [comment]
Thanks!

**Pull request merged**